docs/changes.rst

```diff
@@ -21,6 +21,40 @@ Hypothesis APIs come in three flavours:
 You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
+------------------
+3.7.0 - 2017-03-20
+------------------
+
+This is a feature release.
+
+New features:
+
+* Rule based stateful testing now has an @invariant decorator that specifies
+  methods that are run after init and after every step, allowing you to
+  encode properties that should be true at all times. Thanks to Tom Prince for
+  this feature.
+* The decimals strategy now supports allow\_nan and allow\_infinity flags.
+* There are significantly more strategies available for numpy, including for
+  generating arbitrary data types. Thanks to Zac Hatfield Dodds for this
+  feature.
+
+Bug fixes:
+
+* Bug fix: @composite now preserves functions docstrings.
```

Comment:
function's is also wrong FWIW (it's "functions'" because plural, or "a function's" if you want to keep the singular).